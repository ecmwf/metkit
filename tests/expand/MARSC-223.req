retrieve,         date=2023-12-21,expver=1,grid=0.125/0.125,levtype=sfc,padding=0,param=33.128,step=0,stream=oper,time=00:00:00,type=fc,use=infrequent,
target="reference.zcbyW7.data"