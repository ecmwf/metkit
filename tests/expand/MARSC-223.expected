 RETRIEVE,
            CLASS      = OD,
            TYPE       = FC,
            STREAM     = OPER,
            EXPVER     = 0001,
            REPRES     = GG,
            LEVTYPE    = SFC,
            PARAM      = 33,
            DATE       = 20231221,
            TIME       = 0000,
            STEP       = 0,
            DOMAIN     = G,
            TARGET     = reference.zcbyW7.data,
            RESOL      = AUTO,
            GRID       = .125/.125,
            PADDING    = 0,
            USE        = INFREQUENT 